File: MODUL 6/Bahasa C/Prak601_2410817310006_AhmadLuthfiMaulana.c
#include <stdio.h>

int main() {
    int x, y, rows, cols;

    scanf("%d %d", &rows, &cols);

    int matrix[rows][cols];

    for (x = 0; x < rows; x++) {
        for (y = 0; y < cols; y++) {
            scanf("%d", &matrix[x][y]);
        }
    }

    for (x = 0; x < rows; x++) {
        for (y = 0; y < cols; y++) {
            printf("%d ", matrix[x][y]);
        }
        printf("\n");
    }

    return 0;
}

File: MODUL 6/Bahasa Pyhton/Prak601_2410817310006_AhmadLuthfiMaulana.py
rows, cols = map(int, input().split())
angka=[]
x = 0
isi = list(map(int, input().split()))
for i in range (rows):
    angka.append(isi[x:x+cols])
    x = x + cols
for rows in angka:
    for isi in rows:
        print(isi, end=' ')
    print()